modules/apps/analytics/analytics-web/src/main/java/com/liferay/analytics/web/internal/servlet/taglib/AnalyticsTopHeadJSPDynamicInclude.java

```diff
@@ -176,6 +181,22 @@ private boolean _isAnalyticsTrackingEnabled(
 		return false;
 	}
 
+	private boolean _isCookiesExplicitConsentMode(ThemeDisplay themeDisplay) {
+		try {
+			CookiesPreferenceHandlingConfiguration
+				cookiesPreferenceHandlingConfiguration =
+					_cookiesConfigurationProvider.
+						getCookiesPreferenceHandlingConfiguration(themeDisplay);
+
+			return cookiesPreferenceHandlingConfiguration.explicitConsentMode();
```

Comment:
@marcosapmf let's check here also for the `enabled` flag. Even if `explicitConsentMode` is true, it should take any effect unless the enabled checkbox is also true.

Reply:
<img width="943" alt="Screenshot 2024-03-12 at 3 22 47 PM" src="https://github.com/liferay-ac/liferay-portal/assets/286892/391804af-96c5-47cc-adcb-5de798e29de6">

In this case above, explicitConsentMode should be treated as false.

Reply:
Please replace it to 

`return cookiesPreferenceHandlingConfiguration.enabled() && cookiesPreferenceHandlingConfiguration.explicitConsentMode();`